src/models/review.js

```diff
@@ -1,11 +1,24 @@
-let id = 0;
-
-export default class Review {
-  constructor(productId, author, text) {
-    id += 1;
-    this.id = String(id);
-    this.productId = productId;
-    this.author = author;
-    this.text = text;
-  }
-}
+import mongoose from 'mongoose';
+
+const reviewSchema = mongoose.Schema({
+  _id: { type: String },
```

Comment:
Why not Schema.Types.ObjectId? And I think you missed `unique: true`.

Reply:
Yeah, I might miss this one day